### PR TITLE
fix(accordion): correct header text font weight

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Corrected header text font-weight

## Testing Instructions

1. View Deploy Preview, Safari, Firefox, and Chrome should have the same font-weight.

### CODE REVIEW

File: .changeset/purple-drinks-double.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+<rh-accordion>`: corrected accordion header text font weight
```
Comment (bennypowers) [inline .changeset/purple-drinks-double.md:5]:
```suggestion
`<rh-accordion>`: corrected accordion header text font weight
```

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: .changeset/purple-drinks-double.md (hunk 1 above)
Comment (zeroedin) [reply .changeset/purple-drinks-double.md:5]: oops good catch copy/paste fail.

**Review by marionnegp — COMMENTED** [review]
Heading weights look great in Chrome, Firefox, Safari, and Edge on my Mac.

In Safari, I'm seeing duplicated and empty accordions in the demos. I added a couple screenshots below. Would you want this created as a separate issue?

Here's the default accordion demo:
Screenshot 2024-12-10 at 4 21 38 PM

This is the nested accordion demo (+ crazy icon duplication in the CTA):
Screenshot 2024-12-10 at 4 23 31 PM

**Review by bennypowers — APPROVED** [review]

**Comment by bennypowers** [comment]
@marionnegp that's a separate issue, affects all SSR demos in safari

**Pull request merged**